cypress/page/login.page.ts

```diff
@@ -0,0 +1,21 @@
+class LoginPage {
+    private email: string;
+    private password: string;
+    private submit: string;           
+
+    constructor() {
+        this.email ="#email";
+        this.password ="#passwd";
+        this.submit = "[name=SubmitLogin]";        
+    }
+    
+    public signIn(a :string, b : string): void {
```

Comment:
rename input parameters to something more readable, like userEmail and userPass or something similar

Reply:
the parameters of the method which are cllaed `a` and `b` are the ones that needs to be renamed, not the class variables used as selectors

Reply:
I´m sorry... =(
I misunderstood you!
It is now correct, I hope so...